hack/run-integration-tests.sh

```diff
@@ -17,11 +17,15 @@ fi
 
 if [ "$INTEGRATION_OUTPUT_JUNIT" == true ]
 then
-    echo "Running integration tests with junit output"
+    echo "Running go integration tests with junit output"
     mkdir -p reports/
     go get github.com/jstemmer/go-junit-report
     go test -tags integration ./pkg/... ./cmd/... -v ${MOD_FLAGS} -coverprofile cover-integration.out 2>&1 |tee /dev/fd/2 |go-junit-report -set-exit-code > reports/integration_report.xml
 else
     echo "Running integration tests without junit output"
     go test -tags integration ./pkg/... ./cmd/... -v ${MOD_FLAGS} -coverprofile cover-integration.out
 fi
+
+echo "Running KUTTL integration tests"
+
+./hack/run-kuttl-tests.sh
```

Comment:
It would be better to have this called from the script location and not assume where it is called from... but non-blocking